### ISSUE TITLE
Support -Wp,-D options

### ISSUE THREAD

**Issue by jrosdahl** [ccache/ccache#100]
From https://bugzilla.samba.org/show_bug.cgi?id=11943:

```
Is there any reason ccache couldn't support -Wp,-D options?
```

**Comment by jrosdahl** [comment]
> Is there any reason ccache couldn't support -Wp,-D options?

Nope! It's just that nobody has implemented support for them (yet).

**Comment by jrosdahl** [comment]
Fixed by #102.
